common/src/main/java/com/microsoft/identity/common/internal/cache/SchemaUtil.java

```diff
@@ -259,4 +259,45 @@ public static String getHomeAccountId(final ClientInfo clientInfo) {
 
         return homeAccountId;
     }
+
+    /**
+     * Get tenant id claim from Id token , if not present returns the tenant id from client info
+     * @param clientInfoString : ClientInfo
+     * @param idTokenString : Id Token
+     * @return tenantId
+     */
+    public static String getTenantId(@Nullable final String clientInfoString,
+                                     @Nullable final String idTokenString) {
+
+        String tenantId = "";
```

Comment:
From a behavioral predictability standpoint, let's have this return `null` and declare the method `@Nullable`